src/cdk/testing/protractor/BUILD.bazel

```diff
@@ -14,3 +14,8 @@ ts_library(
         "@npm//protractor",
     ],
 )
+
+filegroup(
+    name = "source-files",
+    srcs = glob(["**/*.ts"]),
```

Comment:
For all of these, should we exclude `**/*.spec.ts`?

Reply:
Yes. technically you are right. Those shouldn't be needed for docs generation, but it will make the +100 `source-files` filegroups more verbose. so maybe fix that up in the future?

Reply:
Somehow it would feel a bit weird adding the exclusion glob here though? I mean, then we have another target in the build file that needs to know about spec files. We should figure something out, but probably not related to this PR.

Reply:
As discussed on Slack. This is not an issue now, nor does it have any big negative effect. Though we should consider cleaning this up in the future: COMP-238